tabular/src/autogluon/tabular/models/tabular_nn/torch/tabular_nn_torch.py

```diff
@@ -562,9 +562,34 @@ def _get_default_resources(self):
         num_gpus = 0
         return num_cpus, num_gpus
 
+    def save(self, path: str = None, verbose=True) -> str:
+        import torch
+        # Save on CPU to ensure the model can be loaded on a box without GPU
+        self.model = self.model.to(torch.device("cpu"))
+        path = super().save(path, verbose)
+        # Put the model back to the device after the save
+        self.model.to(self.device)
+        return path
+
     @classmethod
     def load(cls, path: str, reset_paths=True, verbose=True):
+        import torch
```

Comment:
add brief docstring explaining why this custom `load` method is necessary

Reply:
load was there before